lib/tags/changelog.js

```diff
@@ -18,7 +18,7 @@ module.exports = function changelog (hexo) {
   return globAsync(pathToChangelogs)
   .call('reverse')
   .then((paths) => {
-    const shouldSlice = env !== 'production' | 'staging'
+    const shouldSlice = env !== 'production' || 'staging'
```

Comment:
I didn't catch this the first time around either - but you have two expressions here...

```js
env !== 'production'
// ...and...
'staging'
```

This wil result in 2nd `staging` expression always to be truthy.

Needs to be either... `env === 'production' || env === 'staging'` or something like `['production', 'staging'].includes(env)`

Also cannot use `!==` when comparing two things since one will always be true.